src/taskpane/server.js

```diff
@@ -1926,15 +2140,21 @@ function createEntryFromRow(row, model, fieldTypeEnums, artifactIsHierarchical,
 
   //we need to turn an array of values in the row into a validated object
   for (var index = 0; index < row.length; index++) {
-
+    var skipField = false;
+    if ((isUpdate && fields[index].type == fieldTypeEnums.id) || (isUpdate && fields[index].type == fieldTypeEnums.subId)) {
+      var skipField = false;
+    }

```

Comment:
This if block sets a field already false to false. What it does do however is make sure the else if blocks do not run .
So suggest you change the line "var skipField = false;" to simply "//do nothing"